### PR TITLE
Monit26147-DOTNET-manual instrumentation

### PR DESCRIPTION
**Pull request by deosu**
1. Add a new web application.
2. Update the readme.md file with information about manual instrumentation.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,294 @@
+### Java template
```
Comment (thepeterstone) [inline .gitignore:1]: I'm curious, did you use a set of templates for this?

**Review by thepeterstone — APPROVED** [review]
Looks good to me! Thanks Sumit

File: .gitignore (hunk 1 above)
Comment (deosu) [reply .gitignore:1]: Yes, I am using a plugin provided by Jetbrains.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -92,20 +92,14 @@ jobs:
...
-        run: |
           TEMP_OUTPUT_FILE=$(mktemp)
           HTTP_CODE=$(curl --silent --output $TEMP_OUTPUT_FILE --write-out "%{http_code}" "http://localhost:5114")
           if [[ ${HTTP_CODE} -lt 200 || ${HTTP_CODE} -gt 299 ]] ; then
             >&2 cat $TEMP_OUTPUT_FILE
             exit 1;
           fi
           cat $TEMP_OUTPUT_FILE
-          rm $TEMP_OUTPUT_FILE
+          rm $TEMP_OUTPUT_FILE
```
Comment (thepeterstone) [inline .github/workflows/main.yml]: Looks like some unnecessary whitespace snuck in

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -92,20 +92,14 @@ jobs:
...
         with:
           dotnet-version: ${{ matrix.dotnet-version }}
-      - name: Run the app
+      - name: Run and Test the app
         run: |
           dotnet run --project DOTNET-example/Web-App &
-      - name: Sleep for 30 seconds
-        uses: jakejarvis/wait-action@master
-        with:
-          time: '30s'
```
Comment (thepeterstone) [inline .github/workflows/main.yml]: No need for a sleep with the tasks combined into a single run step? Is there a flaky race condition if the `dotnet run` command doesn't get the app up by the time we run `curl`?

**Review by thepeterstone — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 2 above)
Comment (deosu) [reply .github/workflows/main.yml]: I think these files are outdated.

File: .github/workflows/main.yml (hunk 3 above)
Comment (deosu) [reply .github/workflows/main.yml]: We do not need to sleep. This file is outdated.

**Review by thepeterstone — APPROVED** [review]
LGTM
